components/demo_pages/PaymentPage.module.css

```
@@ -0,0 +1,138 @@
+.container {
+  width: 375px;
+  height: 770px;
+  padding: 14px 25px;
+  background-image: linear-gradient(
+    to bottom,
+    #e2e5ff 1%,
+    rgba(255, 255, 255, 0)
+  );
+  display: flex;
+  flex-direction: column;
+  border: solid 1px lightgray;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 4px;
+}
+
+.wrapper_blue {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 4px;
+  color: #2c46b5;
+}
```

A lot of redundant properties, I think we can do something like this
```suggestion
.wrapper {
  display: flex;
  flex-direction: row;
  align-items: center;
  gap: 4px;
}

.wrapper_blue {
  color: #2c46b5;
}
```